pexpect/pty_spawn.py

```diff
@@ -774,6 +774,11 @@ def sigwinch_passthrough (sig, data):
             p = pexpect.spawn('/bin/bash')
             signal.signal(signal.SIGWINCH, sigwinch_passthrough)
             p.interact()
+        
+        .. note ::
+            Even after Python 3 ``unicode='utf-8'`` support, meth:`interact`
+            doesn't guaranteed to pass input_filter and output_filter str value.
+            You may need to wrap your function to decode and encode back to UTF-8.
```

Comment:
We can be more definite about this: the type is consistent: always bytes, regardless of Python version or `encoding=` argument.

I'd also like the docstring to be improved by updating the paragraph above that talks about filter functions, rather than sticking an extra note on.